### PR TITLE
add video streaming samples

### PR DESCRIPTION
**Pull request by dizcology**
Video Intelligence API streaming samples.

### CODE REVIEW

File: video/cloud-client/analyze/beta_snippets.py [hunk 1]
```diff
@@ -274,6 +288,290 @@ def track_objects(path):
     return object_annotations
 
 
+def detect_labels_streaming(path):
+    # [START video_streaming_label_detection_beta]
+    from google.cloud import videointelligence_v1p3beta1 as videointelligence
```
Comment (nnegrey) [inline video/cloud-client/analyze/beta_snippets.py]: Should add a commented out variable `path` if you don't include the method def.
```
// path = 'path_to_file'

File: video/cloud-client/analyze/beta_snippets.py [hunk 2]
```diff
@@ -274,6 +288,290 @@ def track_objects(path):
...
+
+    # Each response corresponds to about 1 second of video.
+    for response in responses:
+        # Check for errors.
+        if response.error.message:
+            print(response.error.message)
+            break
+
+        for annotation in response.annotation_results.shot_annotations:
+
```
Comment (nnegrey) [inline video/cloud-client/analyze/beta_snippets.py]: nit: empty line

File: video/cloud-client/analyze/beta_snippets.py (hunk 1 above)
Comment (dizcology) [reply video/cloud-client/analyze/beta_snippets.py]: done

File: video/cloud-client/analyze/beta_snippets.py (hunk 2 above)
Comment (dizcology) [reply video/cloud-client/analyze/beta_snippets.py]: done

**Pull request merged**